ITR/data/base_providers.py

```diff
@@ -36,6 +36,9 @@ def __init__(self, production_benchmarks: IProductionBenchmarkScopes,
         self.temp_config = tempscore_config
         self.column_config = column_config
         self._productions_benchmarks = production_benchmarks
+        # production benchmarks are input for S1S2, but relevant for all scopes
+        self._productions_benchmarks.S3 = self._productions_benchmarks.S1S2
```

Comment:
I know that you didn't like the idea that a reported S3 emission could become an S1 emissions under the "Producer-centric" benchmark model.   By that same token, to copy S1S2 scope emissions in an S3 container, you need to show which benchmark says it's ok under what conditions.  S1S2 means S1+S2.  S1S2S3 means S1+S2+S3, so there are cases where you could put S1S2 into S1S2S3 (as long as you add S3).  But I don't think you can copy S1S2 into S3 by itself.